app/policies/contest_group_policy.rb

```diff
@@ -0,0 +1,31 @@
+class ContestGroupPolicy < ApplicationPolicy
+  def index?
+    admin_or_coach?
+  end
+
+  def new?
+    admin_or_coach?
+  end
+
+  def create?
+    admin_or_coach?
+  end
+
+  def edit?
+    admin_or_coach?
+  end
+
+  def update?
+    admin_or_coach?
+  end
+
+  def destroy?
+    admin_or_coach?
+  end
+
+  private
+
+  def admin_or_coach?
+    user && (user.admin? || user.coach?)
+  end
+end
```

Comment:
Please, configure your editor to add a new line at the end of the files. This is a convention followed in unix: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline